kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -414,51 +388,11 @@ class SolvingStrategy
         {
             "name"                         : "solving_strategy",
             "move_mesh_flag"               : false,
-            "echo_level"                   : 1,
-            "build_level"                  : 2
+            "echo_level"                   : 1
         })");
         return default_parameters;
     }
   
-    /**
-     * @brief This method returns the LHS matrix
-     * @return The LHS matrix
-     */
-    virtual TSystemMatrixType& GetSystemMatrix()
-    {
-        KRATOS_TRY
-
-        KRATOS_ERROR << "GetSystemMatrix not implemented in base SolvingStrategy" << std::endl;
-
-        KRATOS_CATCH("");
-    }
-
-    /**
-     * @brief This method returns the RHS vector
-     * @return The RHS vector
-     */
-    virtual TSystemVectorType& GetSystemVector()
-    {
-        KRATOS_TRY
-
-        KRATOS_ERROR << "GetSystemVector not implemented in base SolvingStrategy" << std::endl;
-
-        KRATOS_CATCH("");
-    }
-
-    /**
-     * @brief This method returns the solution vector
-     * @return The Dx vector
-     */
-    virtual TSystemVectorType& GetSolutionVector()
```

Comment:
this was only recently added: #6763
I think it cannot be removed

Reply:
It is added in the implicit strategy. It makes no sense to have it in the base one as it's meaningless in the explicit one.

Reply:
I know, but the Factories will give most probably the baseclasses
and if those fcts are missing in the baseclass they cannot be accesses

I agree that they ake no sense in explicit but sometimes it is necessary (e.g. the B&S baseclass also has some fcts that don't make sene for Block or ElimB&S)

Reply:
Agree and done. When doing this I didn't think about the factories. Thanks for pointing it out.